sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubApplicationInsightsTests.cs

```diff
@@ -348,7 +348,7 @@ public class EventHubTestMultipleDispatchJobs
 
             private const int EventCount = 5;
             private static readonly object MessageLock = new object();
-            public static void SendEvents_TestHub(string input, [EventHub(TestHubName)] out EventData[] events)
+            public static void SendEvents_TestHub(string input, [EventHub(TestHubName, Connection = TestHubName)] out EventData[] events)
```

Comment:
Do we have a test using ConsumerGroup?

Reply:
Don't think so

Reply:
Consider adding one - you can create an EH with a consumer group in the existing EventHubScope

Reply:
OK

Reply:
I added a test that uses CG passed through the configuration.